scss/login.scss

```diff
@@ -0,0 +1,88 @@
+@mixin form-element {
+  width: 100%;
+  border-radius: 5px;
+  height: 32px;
```

Comment:
is this height necessary?

Reply:
It is, if I don't specify it the elements take a height of 16px which is the font-size